vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/credentialverifier/CredentialVerifierFactory.kt

```diff
@@ -1,14 +1,12 @@
 package io.mosip.vercred.vcverifier.credentialverifier
 
 import io.mosip.vercred.vcverifier.constants.CredentialFormat
-import io.mosip.vercred.vcverifier.credentialverifier.types.LdpVcCredentialVerifier
+import io.mosip.vercred.vcverifier.credentialverifier.types.LdpVerifiableCredential
 
 class CredentialVerifierFactory {
-    fun verify(credential: String, credentialFormat: CredentialFormat): Boolean {
+    fun get(credentialFormat: CredentialFormat): LdpVerifiableCredential {
```

Comment:
Should we return VerifiableCredential? instead of LdpVerifiableCredential for  extensibility?

Reply:
yes, it should return VerifiableCredential

Reply:
yes, might have missed it. Will update it